src/main/scala/com/evolutiongaming/retry/OnError.scala

```diff
@@ -31,7 +31,7 @@ object OnError {
 
       decision match {
         case OnError.Decision.Retry(delay) =>
-          log.warn(s"failed, retrying in $delay, error: $error")
+          log.warn(s"failed, retrying in $delay, error: $error", error)
```

Comment:
why do you think this is a good idea to print stacktrace along with warn?

Reply:
https://github.com/evolution-gaming/retry/pull/159#issuecomment-1354685358

Reply:
we are talking here about default behaviour which you can override as user of the lib.
I believe that you don't need stack traces in cases when error was successfully retried and in case it was failed - you will have stacktrace. Wdyt?

Reply:
I think that having full picture of failure (even if it was retried) is useful. Widely used log aggregation systems provide great UI that allow one to hide/expand error details and let one choose between different overview options ... but only while full stack-traces are present.

I strongly against using `Throwable#toString` anywhere and don't want to redefine so common lib component as `OnError` everywhere.